src/FeatureIntroControl/InteractiveFeatureIntro.tsx

```diff
@@ -0,0 +1,60 @@
+import { ClassNames } from "@emotion/core";
+import React from "react";
+import * as typography from "../typography";
+import { colors } from "../colors";
+import Tippy from "@tippyjs/react";
+import { FeatureIntroControl } from "../FeatureIntroControl";
+import { FeatureIntroContent } from "../FeatureIntroContent";
+import { FeatureIntroHeading } from "../FeatureIntroHeading";
+import { FeatureIntroLearnMoreLink } from "../FeatureIntroLearnMoreLink";
+import { FeatureIntroDismissButton } from "../FeatureIntroDismissButton";
+import { FeatureIntroTippyStyles } from "./FeatureIntroTippyStyles";
+
+export const InteractiveFeatureIntro: React.FC = () => {
```

Comment:
This component looks like it maybe belongs in studio code to me?

Reply:
This was more of a test for me to make sure things work since I had a lot of issues npm linking. I'm happy to delete it, but I also think it shows how the FeatureIntro is supposed to work (in a Popover). Let me know!